content/git/concepts/reset/reset.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Reset'
+Description: 'In Git, reset is used to set the HEAD pointer back to a previous commit and optionally undo staged changes and the working tree.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the `reset` command is used to change the state of the Git repository or undo commits.
+
+## Syntax
+
+In the terminal, the "git reset" command can be used by typing the following:
+
+```pseudo
+git reset <mode-option> <commit-reference-number>
+```
+
+### Mode Options
+
+The `<mode-options>` refers to how far `reset` will go when rolling back changes to a previous commit, including:
+
+- Where the `HEAD` is pointing towards (usually done with just `git reset`).
+- Whether the staging area or Index, reflects the commit the `HEAD` is now pointing towards.
+- Whether the working tree is rolled back to reflect the changes reset in the `HEAD` and staging area.
+
+More specifically, these options include:
+
+1. --soft: This undoes the commit but still shows the file as staged and permits only new commits.
+2. --mixed: This undoes the commit and unstages the file and the file can be edited and committed normally.
+3. --hard: This removes the new commit including all changes and uncommitted files.
+
+The commit can either be the commit hash generated by Git, or in reference to the HEAD.
+
+## Example
+
+Suppose a file named **example.txt** is accidentally committed to a Git repository. To undo the commit, the command `git reset --mixed HEAD~1` is used in the terminal. `--mixed` is the default mode, so `git reset HEAD~1` will achieve the same result. This resets the HEAD and index to the previous commit.
+
+**Note:** HEAD refers to the current commit while index refers to the staging area where files are [added using Git](https://www.codecademy.com/resources/docs/git/add).
```

Comment:
This is something that should be mentioned closer to the top of the entry (probably in the "## Syntax" section)

Reply:
> This is something that should be mentioned closer to the top of the entry (probably in the "## Syntax" section)

I thought since we were showing an example that it would be ok, giving a context to the example under the ## Example instead of near the top, I can still change it if y'all want me to.

Reply:
The example is great! 😃 I'm only suggesting moving this part of the text because `HEAD` is first mentioned in the "## Syntax" anyway but this seems to define `HEAD` ("refers to the current commit"). `HEAD` alls under a possible value for the `commit-reference-number` used in the syntax code. I'll leave a suggestion up top to show what I mean.

Reply:
> The example is great! 😃 I'm only suggesting moving this part of the text because `HEAD` is first mentioned in the "## Syntax" anyway but this seems to define `HEAD` ("refers to the current commit"). `HEAD` alls under a possible value for the `commit-reference-number` used in the syntax code. I'll leave a suggestion up top to show what I mean.

Hey sure, so if I'm not wrong you mean to push up line 45 right?

I did not understand what you meant by 
> HEAD alls under a possible value for the commit-reference-number used in the syntax code.

Reply:
@Dusch4593